src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -39,11 +42,26 @@
     public static final String MESSAGE_SUCCESS_STATISTICS = "Listed the statistics for the day";
     public static final String MESSAGE_SUCCESS_HIGH_RISK_LOCATIONS = "Listed high risk locations";
     public static final String INVALID_LIST_TYPE = "There is no such list type.";
+    public static final String INVALID_HIGH_RISK_LOCATIONS_NUMBER = "Number for high risk locations should be a"
+            + " non-negative integer and should not be larger than number of total locations.";
```

Comment:
Small grammar mistake. Instead of "number of total locations.", perhaps it should be "the total number of locations."

Reply:
Fixed.